.dependabot/config.yml

```diff
@@ -0,0 +1,127 @@
+version: 1
+
```

Comment:
Shouldn't we specify the [`commit_message`](https://dependabot.com/docs/config-file/#commit_message) too like we did with Greenkeeper? What is the default commit message if we don't specify one?

Reply:
Actually, we cannot customize commit messages as we can with GreenKeeper.
It's quite limited. The options are available here: https://dependabot.com/docs/config-file/#commit_message 

You can see below the commit messages it does.

#### Update of a dep

```txt
chore(deps): bump @babel/preset-env in /packages/stark-build

Bumps [@babel/preset-env](https://github.com/babel/babel) from 7.7.1 to 7.8.2.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/master/CHANGELOG.md)
- [Commits](babel/babel@v7.7.1...v7.8.2)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>
```
#### Auto merge of a PR:

```txt
Merge pull request #118 from SuperITMan/dependabot/npm_and_yarn/packages/stark-build/babel/preset-env-7.8.2
```

Reply:
Ok, so maybe just specify this:

```
commit_message:
      prefix: "chore"
      include_scope: true
```

to ensure that the commit message will always be: `chore(deps): bla bla` (I know that's the default but just in case 😉 )

Reply:
Change applied :blush: